### DISCUSSION TITLE
PlaceholderAPI work with menu title placeholder

### DISCUSSION THREAD

**Discussion by samcefalo** [BG-Software-LLC/SuperiorSkyblock2#341] [Implemented Feature Requests] [answered]
Placeholders provided by menu titles do not work with PlaceholderAPI, they are loaded after loading placeholderapi, eg I can't use %formatter_number_{1}% in titles, placeholder {1} is loaded later. I checked the source and really, the values ​​menu for example loads placeholder {0} and {1} after having loaded PlaceholderAPI

**Comment by OmerBenGera** [comment, marked as answer]
Thanks for making a PR :)
